leo/src/app/vendors/page.tsx

```diff
@@ -8,13 +8,19 @@ const vendors = [
         category: "Catering",
         contact: "contact@vendor-a.com",
         events: ["Wedding", "Corporate Party", "Concert"],
+        created_at: '1/1/2024',
+        phone_number: 123456789,
+        email: 'test@gmail.com'
```

Comment:
add that [stack overflow](https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression) regex to check if it is a valid email

Reply:
I don't think we should add that here. We should add it to the form validation when we create the vendor admin page and when we update the event admin page (where they would be adding and updating their email addresses)